app/decorators/quiz_decorator.rb

```diff
@@ -0,0 +1,11 @@
+class QuizDecorator < Draper::Decorator
+  delegate_all
+
+  def student_column_name
+    object.name.split.first.downcase + '_quiz_completed'
+  end
```

Comment:
We want this method in the Quiz model, not the decorator, since we need it for `Student#quiz_completed?`. In general, we can only assume the decorator methods are available in the views (meaning we shouldn't call this method in a model).

For example, let's suppose you forgot to decorate a Quiz object before calling `student.quiz_completed? quiz`. This would error because the quiz object doesn't have the `student_column_name` method. Because decorators are only supposed to be used for views, it's bad practice for us to have to decorate the quiz every time we need to call `quiz_completed?`.